Android.mk

```diff
@@ -116,4 +116,5 @@ LOCAL_MODULE_CLASS := SHARED_LIBRARIES
 LOCAL_MODULE_SUFFIX := $(TARGET_SHLIB_SUFFIX)
 include $(BUILD_SHARED_LIBRARY)
 
+#include $(LOCAL_PATH)/tests/third_party/json-c/Android.mk
```

Comment:
hmm I think we can leave it uncommented. If we dont want to have tests part of user builds, than can include this only for eng builds. For user builds if someone wants it than they can uncomment it.